TreeTracker/UI/Views/Home/HomeViewController.swift

```diff
@@ -99,6 +100,24 @@ private extension HomeViewController {
     }
 }
 
+// MARK: - Private
+private extension HomeViewController {
+    func profileButton(withImageData data: Data) {
```

Comment:
Maybe something like `updateProfileButton` could make the function name here more descriptive?

I think `profileButton` sounds like it should return a button.

What do you think?